packages/mobile/src/components/details-tile/DetailsTileNoAccess.tsx

```diff
@@ -30,12 +115,168 @@ export const DetailsTileNoAccess = ({
   trackId,
   premiumConditions
 }: DetailsTileNoAccessProps) => {
+  const styles = useStyles()
   const premiumTrackStatusMap = useSelector(getPremiumTrackStatusMap)
   const premiumTrackStatus = premiumTrackStatusMap[trackId] ?? null
 
-  if (premiumTrackStatus === 'UNLOCKING') {
-    return <DetailsTileUnlockingSection premiumConditions={premiumConditions} />
-  }
+  const { nftCollection, followee, tippedUser } =
+    useSpecialAccessEntity(premiumConditions)
+
+  const collectionLink = useMemo(() => {
+    if (!nftCollection) return ''
+
+    const { chain, address, externalLink } = nftCollection
+    if (chain === Chain.Eth && 'slug' in nftCollection!) {
+      return `https://opensea.io/collection/${nftCollection.slug}`
```

Comment:
Also if logic is the same, should we pull it out to a separate function?

Reply:
fair